pants.toml

```
@@ -111,38 +111,36 @@ enable_resolves = true
 default_resolve = "st2"
 interpreter_constraints = [
   # python_distributions needs a single constraint (vs one line per python version).
-  "CPython>=3.8,<3.10",
+  "CPython==3.9.*",
```

This is the interpreter constraints for our code, so this said that st2 only supports py 3.9.

```suggestion
  "CPython>=3.8,<3.11",
```

---

That probable explains why I couldn't get pants to run without py3.9